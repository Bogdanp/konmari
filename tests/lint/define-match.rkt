#lang racket/base

(require racket/match)

(define/match (f x)
  [(1)
   (define x+1 (add1 x))
   x+1]
  [(2)
   (define x+1 (add1 x))
   x+1])

(f 1)
