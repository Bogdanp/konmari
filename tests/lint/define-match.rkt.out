/define-match.rkt:5:18:warning:identifier 'x' is never used
